package.json

```diff
@@ -16,6 +16,12 @@
       "**/@types/theme-ui__components"
     ]
   },
+  "preconstruct": {
+    "packages": [
+      "packages/*",
+      "tooling/cli"
```

Comment:
Since `cli` is the only package in this folder that's actually built, I didn't add a wildcare for the `tooling` directory. I'm wondering if `cli` should be moved into `packages` since it's an official part of the Chakra API.